### PR TITLE
Load more button indicate loading

### LINKED ISSUE #179: 'Load More'-button does not show loading
## Description
The 'load more' in pagination-setting should indicate loading-state

### PR DESCRIPTION
**Pull request by tordjon**
Closes #179 

__EDIT__: added issue reference - @olaven

### CODE REVIEW

**Review by olaven — COMMENTED** [review]
😄 Code appears to fit in quite nicely with the existing structure/style. Seems to work great on the page list. `TextList`is broken, but Ill see if I can figure it out. 
🎉

**Comment by olaven** [comment]
The error was caused by what seems to be a mixup between "a page created by the user" and "a page as in pagination". 

I think this indicates that the frontend-"arcitecture" for fetching paginaged endpoints should be rethought.
1. confusing naming 
2. a lot of "intermediate components" introducing (perhaps?) needless complexity 
3. a lot of duplication between these "intermediate components", and new ones would have to be added for each new endpoint/concept being fetched.

File: src/effects/usePagination.tsx [hunk 1]
```diff
@@ -29,22 +29,22 @@ export const usePagination = function <T>(basePath: string): [
...
     asyncEffect(async () => {
+
         setPageLoading(true)
+
         const page = await filterBody(
             get<PaginatedModel<T>>(next)
         );
-        await new Promise(resolve => setTimeout(resolve, 2000)) //TODO: REMOVE
+
+        await new Promise(resolve => setTimeout(resolve, 2000))
```
Comment (tordjon) [inline src/effects/usePagination.tsx]: 😅

Comment (olaven) [reply src/effects/usePagination.tsx]: 😄

**Pull request merged**